zabbix_cli/cli.py

```diff
@@ -6765,15 +6829,15 @@ def do_load_balance_proxy_hosts(self, args):
 
         #
         # Getting all host monitored by the proxies defined in
-        # proxyid_list. These are the host that will get spreaded
-        # evenly along the defined proxies.
+        # proxy_list.  These are the host that will get spread
+        # along the defined proxies.
```

Comment:
Should this perhaps be "spread among" rather than "spread _along_"?

Reply:
Indeed; I used "among" elsewhere in the code, but my brain apparently struggles to separate the concepts on the fly.  Good catch :)